### PR TITLE
Remove commented out code and redundant comments from std.experimental.allocator

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/experimental/allocator/building_blocks/null_allocator.d [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
 
-/*
-  _   _       _ _          _ _                 _
- | \ | |     | | |   /\   | | |               | |
- |  \| |_   _| | |  /  \  | | | ___   ___ __ _| |_ ___  _ __
- | . ` | | | | | | / /\ \ | | |/ _ \ / __/ _` | __/ _ \| '__|
- | |\  | |_| | | |/ ____ \| | | (_) | (_| (_| | || (_) | |
- |_| \_|\__,_|_|_/_/    \_\_|_|\___/ \___\__,_|\__\___/|_|
-*/
-
```
Comment (wilzbach) [inline std/experimental/allocator/building_blocks/null_allocator.d]: That's funny - any reason why we shouldn't keep it?

Comment (ZombineDev) [inline std/experimental/allocator/building_blocks/null_allocator.d]: +1 :D

Comment (JackStouffer) [inline std/experimental/allocator/building_blocks/null_allocator.d]: It's redundant

Comment (wilzbach) [inline std/experimental/allocator/building_blocks/null_allocator.d]: sure, but it lets a couple of people smile if they discover this in the source code.
I don't really mind though ;-)

**Comment by JackStouffer** [comment]
Ping

**Comment by JackStouffer** [comment]
Ping, this is trivial @burner @schveiguy @Hackerpilot

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**